examples/OfflineLogin.md

```diff
@@ -0,0 +1,90 @@
+# Offline synced realm in the .NET Realm SDK
+
+With MongoDB Realm offline-first capabilities, it is always possible to read and write to the database even when using Device Sync, independently from the connection status of the app. In this small article, we are going to see how to open a synced realm depending on the situation in the .NET Realm SDK
+
+There are two methods in the .NET Realm SDK that can be used to open a Realm: `Realm.GetInstance` (synchronously) and `Realm.GetInstanceAsync` (asynchronously). The two methods not only differ in asynchronicity, but have a different behavior when used to open a synced realm. We will first take a look at those two methods on their own, then show a recommended flow for opening a realm. 
+
+#### GetInstance
```

Comment:
[q] Why are these h4? Seems kind of weird going from h1 to h4 with nothing in between.